PlusServerLauncher/PlusServerLauncherMainWindow.cxx

```diff
@@ -346,6 +378,29 @@ void PlusServerLauncherMainWindow::ParseContent(const std::string& message)
   }
 }
 
+//----------------------------------------------------------------------------
+PlusStatus PlusServerLauncherMainWindow::SendCommand(std::string device_id, std::string command, std::string content, igtl::MessageBase::MetaDataMap metaData)

```

Comment:
Does this function send the command to all connected clients? If so, this needs to be configurable as some clients won't want the log spam.

Reply:
The current server doesn't seem to be able to connect to multiple clients at a time, so I haven't been able to test this. My suspicion is that it would.

Reply:
Oh that's an oversight. Would you be willing to add an issue for that? I don't even know if that should go in this project or in igtlio...

Reply:
> The current server doesn't seem to be able to connect to multiple clients at a time

If this is the case then it should be fixed soon. That's a very fundamental limitation.

Reply:
Made an issue in PlusToolkit/PlusLib#350